changes/23078-allow-skipping-vuln-details

```diff
@@ -0,0 +1 @@
+* Allowed skipping computationally heavy population of vulnerability details when populating host software on hosts list endpoint when using Fleet Premium
```

Comment:
Add the API endpoint and new value for argument here?

Reply:
Revised in https://github.com/fleetdm/fleet/pull/23710/commits/210bbed143d2554b28e9fb759862bc69f9554b42